---
bool success
float32 spdPan
float32 spdTilt
float32 accPan
float32 accTilt